package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "eslint src",
     "flow": "flow src",
     "fix-xcode": "react-native-schemes-manager all",
-    "postinstall": "yarn fix-xcode",
+    "postinstall": "yarn fix-xcode || ./scripts/google-maps-ios.sh",
```

Comment:
I am not sure if `||` is the right thing to do here. There are multiple ways to chain shell commands:

* `a || b`: Execute `a`. If this fails (has non zero exit status), execute `b`.
* `a && b`: Execute `a`. If this succeeds (has zero exit status), execute `b`.
* `a; b`: Execute `a`, then execute `b`.

I assume `yarn fix-xcode` succeeds normally. This means with `||` the `google-maps-ios.sh` script will not run. We should probably use `;` instead. It doesn't matter if `yarn fix-xcode` fails or succeeds. We always want to run the maps script.

Reply:
Thanks Jan, I wasn't sure of the syntax here, I will make that change now

Reply:
Yup my bad with the syntax suggestion

Reply:
Np dude
I'm now getting permissions errors with this, any suggestions as to how I would go about making this executable once pushed?

Reply:
Thats a good sign, because it means it tries to run the script. Git stores file permissions (read, write, execute). So you need to make the script executable locally and then commit this change.

You can make it executable by running in a terminal:

```sh
chmod +x ./scripts/google-maps-ios.sh
```

Reply:
Oh wow for some reason I didn't realise that permissions I made locally would persist once pushed. This is aces.